### PR TITLE
Drag skill

### PR DESCRIPTION
**Pull request by wmcclinton**
This PR adds a Drag skill that moves the platform near another object. The drag sampler returns an (x, y) target location of the platform.

### CODE REVIEW

File: predicators/spot_utils/spot_utils.py
Comment (nkumar-bdai) [inline predicators/spot_utils/spot_utils.py:1301]: this isn't an impedance controller anymore right? let's update this comment accordingly

**Review by nkumar-bdai — COMMENTED** [review]
Looks pretty good to me overall, modulo a couple minor changes! Great work. Will wait for it to be working before giving final approval

File: predicators/ground_truth_models/spot/nsrts.py
Comment (wmcclinton) [reply predicators/ground_truth_models/spot/nsrts.py:30]: added

**Review by nkumar-bdai — APPROVED** [review]
glad to see this working!

**Pull request merged**
